fdbserver/DataDistribution.actor.cpp

```diff
@@ -3908,13 +3908,16 @@ ACTOR Future<Void> storageServerTracker(
 			}
 		}
 	} catch( Error &e ) {
-		if (e.code() != error_code_actor_cancelled && errorOut.canBeSet())
+		state Error err = e;
+		if (e.code() != error_code_actor_cancelled && errorOut.canBeSet()) {
 			errorOut.sendError(e);
+			wait(delay(0)); // Check for cancellation
+		}
 		TraceEvent("StorageServerTrackerCancelled", self->distributorId)
```

Comment:
This is added in 6.3 branch. Can you make this against 6.3 branch?

Reply:
Done